app/src/components/BaseHeader.vue

```diff
@@ -11,9 +11,14 @@
     <!--optional by address last seen / last  updated  -->
     <div v-if="formattedOwner">
       <span class="text-grey-400 mr-4">by</span>
-      <a class="link mr-4" :href="`https://etherscan.io/address/${owner}`" target="_blank" rel="noopener noreferrer">{{
-        formattedOwner
-      }}</a>
+      <a
+        v-if="owner"
+        class="link mr-4"
+        :href="getEtherscanUrl(owner, 1, 'address')"
```

Comment:
Can we pull the chainId from the wallet store instead of hardcoding it? You can add a computed property that gets it and exports it, based off the `network` property. Otherwise we'll have to remember to update the hardcoded chainIds here as part of a potential move to an L2

Reply:
done